index.html

```diff
@@ -1,1642 +1,232 @@
+    <p></p>
+
+    <section id="sec-pages-components">
+      <h3>MiniApp Pages and Components</h3>
+      <p>
+        A <dfn>MiniApp Component</dfn> is a custom MiniApp element with encapsulated functionality, data and styles that
+        can be reused in <a href="https://www.w3.org/TR/miniapp-packaging/#dfn-page" data-link-type="dfn">MiniApp Pages</a>.
+      </p>
+      <div class="ednote">
+        Structure, differences with Web Pages and Web Components.
+      </div>
+    </section>
 
-    <section id="conformance">
-    </section>      
+    <section id="sec-interface">
+      <h3>MiniApp Interface</h3>
+      <p>
+      </p>
+      <div class="ednote">
+        Formal definition of the interface?
+      </div>
+    </section>
   </section>
-  
-  <section>
-    <h2>MiniApp UI Components Container Component</h2>
-    <section>
-        <h2>div</h2>
-        <p>The 'div' component is a basic container that is used as the root node of the page structure or is used to group the content.</p>
-        <p>The 'div' component has no additional attributes except the common attributes.</p>
-		<p>The 'div' component not support additional events except the common events.</p> 
 
+  <section id="sec-templating">
+    <h2>Templating</h2>
+    <p></p>
+    <section id="sec-data-interpolation">
+      <h3>Content Interpolation</h3>
+      <p></p>
+      <div class="ednote">
+        Data binding.
+      </div>
     </section>
-    <section>
-        <h2>list</h2>
-        <p>The 'list' component has no additional attributes except the common attributes.</p>
-		<p>In addition to common events, the <list> component supports the following private events as well:</p>
+  </section>
+
+  <section id="sec-declaration">
+    <h2>Component Declaration</h2>
+    <p></p>
+    <div class="ednote">
+      Component interface (properties)
+    </div>
+  </section>
 
-      <table class="members">
-        <caption>Events for list component</caption>
-        <thead>
-          <tr>
-            <th scope="col">Name</th>
-            <th scope="col">Parameter</th>
-            <th scope="col">Description</th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr>
-            <th scope="row"><code>Scrollend</code></th>
-            <td></td>
-            <td>
-              <span>Triggered when the list stops scrolling.</span>
-            </td>
-          </tr>			  
-		</tbody>
-	  </table>
+  <section id="sec-reuse">
+    <h2>Component Reuse</h2>
+    <p>
+      A component may use other custom components registered and defined within the package. For that, developers must import the custom component in the parent component declaration. The imported component is exposed to the component template. It will be available as another tag in the template, identified by a unique key.
+    </p>
+    <pre class="example" title="Import of custom component"><code>&lt;!-- my-component.html --&gt;
+
+&lt;import src="./cool-button" name="cool-button" /&gt;

```

Comment:
Is "./cool-button" a menu or an HTML?

Reply:
It's an HTML. The entry of the imported component.